pages/common/vectorize-pixelart.md

```diff
@@ -0,0 +1,12 @@
+# vectorize-pixelart
+
+> Convert raster pixel art graphics to vector.
+> More information: <https://github.com/und3f/vectorize-pixelart>.
+
+- Vectorize a PNG image to an SVG vector:
+
+`vectorize-pixelart {{image.png}} {{output.svg}}`
+
+- Vectorize to an EPS vector:
+
+`vectorize-pixelart {{image.png}} {{output.eps}}`
```

Comment:
To be fair, as both of these do the same thing (just with different output formats) and don't have any flags, we can probably do a single example with something like the following:
```suggestion
- Convert a PNG image to a vector image format:

`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`
```

Reply:
What about `{{path/to/output.svg|.eps}}` (with `|.eps`)?

Reply:
I'm not entirely sure what our standard is for multiple extensions.